CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ option(USE_OPENCL "Build OpenCL GPU Framework" OFF)
 option(USE_DPDK "Enable DPDK Framework" OFF)
 option(USE_HDF5 "Build HDF5 output stages" OFF)
 option(USE_OMP "Enable OpenMP" OFF)
+option(USE_OLD_ROCM "Build for ROCm versions 2.3 or older" OFF)
```

Comment:
Please add this new option to the README.md and the build documentation next to the current HSA options.